spec.yaml

```diff
@@ -50,6 +52,42 @@ securityDefinitions:
 
 
 paths:
+  /v4/users/:
+    get:
+      operationId: getUsers
+      tags:
+        - users
+      summary: Get users
+      description: |
+        An admin only endpoint that returns a list of all users in the system.
```

Comment:
I could imagine opening this endpoint to non-admin users as well. It could then list all users the requesting user shares an organization with. But not now.

If we agree that this is likely, we should rephrase this along the lines of "Currently this endpoint/function is only available to users with admin permissions".